context/AuthHelper.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { account } from '../api/config';
+import Alert from '../components/AlertNotification/AlertNotification';
+import { AlertVariants } from '../components/AlertNotification/Alerts.enum';
+import { AppRouterInstance } from 'next/dist/shared/lib/app-router-context.shared-runtime';
+
+import { logoutHandler } from './AuthHelper';
+
+import { toast } from 'react-hot-toast';
+
+jest.mock('../api/config');
+jest.mock('next/router');
+jest.mock('react-hot-toast', () => ({
+  toast: {
+    custom: jest.fn(),
+  },
+}));
+
+describe('LogoutHandler', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const router = {
+    push: jest.fn(),
+  } as unknown as AppRouterInstance;
+
+  const resetUser = jest.fn();
+  const setIsSignedIn = jest.fn();
+
+  //logout tests
```

Comment:
Remove the comments from this file. No need for them.

Reply:
fixed